lib/client/ippool_e2e_test.go

```diff
@@ -237,7 +233,7 @@ var _ = Describe("IPPool tests", func() {
 		// Test 5: Test starting with IPIP (cross subnet mode) and moving to no IPIP
 		Entry("IPIP (cross subnet mode) and moving to no IPIP",
 			api.IPPoolMetadata{CIDR: testutils.MustParseNetwork("10.0.0.0/24")},
-			api.IPPoolMetadata{CIDR: testutils.MustParseNetwork("fe80::00/120")},
+			api.IPPoolMetadata{CIDR: testutils.MustParseNetwork("10.10.10.0/24")},
```

Comment:
oh for some reason i thought we swapped Specs with the two pools - maybe we don't.  It's not really testing anything new though having the IPv6 CIDR so I'm ok with it remaining like this - we do have another test that explicitly tests IPv6 CIDR.